src/sources/file.rs

```diff
@@ -250,6 +250,12 @@ pub struct FileConfig {
     #[configurable(derived)]
     #[serde(default)]
     internal_metrics: FileInternalMetricsConfig,
+
+    /// How long to keep an open handle to a rotated log file.
```

Comment:
For here and in the Kubernetes Logs source, can you add an explanation that the default value effectively represents no limit?